src/Symfony/Component/Messenger/Stamp/ErrorDetailsStamp.php

```diff
@@ -32,19 +32,38 @@ final class ErrorDetailsStamp implements StampInterface
     /** @var FlattenException|null */
     private $flattenException;
 
-    public function __construct(Throwable $throwable)
+    /**
+     * @param int|mixed $exceptionCode
+     */
+    public function __construct(
+        string $exceptionClass,
+        $exceptionCode,
```

Comment:
`int`?

Reply:
It was documented as `int|mixed`, probably for the same reason as https://github.com/vimeo/psalm/pull/3175